src/system/armor.hh

```diff
@@ -162,7 +162,7 @@ public:
     inline Mat<Type,nr,nc> get(uint i) const
     {
         ASSERT_DBG( (nr == nRows) && (nc == nCols) );
-        return Mat<Type,nr,nc>( const_cast<Type*>(data.data()) + i*nRows*nCols );
+        return Mat<Type,nr,nc>( (Type *)data.data() + i*nRows*nCols );
```

Comment:
We will probably need both const and non-const get method to allow assignenment.

Reply:
This method already allows assignment. Do we need some other version?